rllib/evaluation/rollout_worker.py

```diff
@@ -897,14 +898,21 @@ def apply_gradients(self, grads: ModelGradients) -> Dict[PolicyID, Any]:
         if log_once("apply_gradients"):
             logger.info("Apply gradients:\n\n{}\n".format(summarize(grads)))
         if isinstance(grads, dict):
-            if self.tf_sess is not None:
-                builder = TFRunBuilder(self.tf_sess, "apply_gradients")
-                outputs = {
-                    pid: self.policy_map[pid]._build_apply_gradients(
-                        builder, grad)
-                    for pid, grad in grads.items()
+            if self.policy_config.get("framework") == "tf":
```

Comment:
This was buggy code: Each policy nowadays has its own tf-graph (instead of the workers/trainer carrying the graph), so we have to loop through each policy here.